travis/travis_publish.sh

```diff
@@ -4,8 +4,8 @@ if [[ $TRAVIS_BRANCH == 'master' ]] && [ "$TRAVIS_PULL_REQUEST" = "false" ]; the
   source travis/extract.sh
   source travis/docker.sh
   sbt "+ publishSigned" \
-      "mleap-xgboost-runtime/publishSigned" \
-      "mleap-xgboost-spark/publishSigned" \
+      "+ mleap-xgboost-runtime/publishSigned" \
+      "+ mleap-xgboost-spark/publishSigned" \
```

Comment:
same as above, these can be removed if we add the mleap-xgboost-runtime and mleap-xgboost-spark subprojects in the aggregatedProjects list in MleapProject.scala`.